ds_judgements_public_ui/sass/includes/_cards_homepage.scss

```diff
@@ -1,89 +1,208 @@
 .cards-homepage {
   display: flex;
-  flex-direction: column;
-  gap: 0;
+  flex-direction: row;
+  gap: $space-4 !important;
 
-  margin: $space-4 auto;
+  margin: 0;
+  margin-top: $space-10;
   padding: 0;
 
   list-style-type: none;
 
+  &:hover {
+    text-decoration: none;
+  }
+
   li {
-    @media (max-width: $grid-breakpoint-medium) {
-      height: 115px;
-      margin: 0 auto;
+    @media (width <= 1199px) {
+      height: $space-8;
+      margin: 0;
+      margin-bottom: $space-5;
+    }
+  }
+
+  &__container {
+    @include container;
+
+    display: flex;
+    flex-direction: column;
+    gap: $space-4;
+    padding-top: $space-8;
+
+    h2 {
+      display: none;
+
+      @media (width <= 1199px) {
+        @include heading_text;
+        @include header($typography-header-h2-size);
+
+        display: contents;
+      }
+
+      @media (max-width: $grid-breakpoint-small) {
+        display: contents;
+        margin-top: 0;
+      }
+    }
+
+    @media (width <= 1199px) {
+      gap: 0;
+      align-items: flex-start;
+
+      margin-top: $space-5;
+      margin-right: $space-5;
+      margin-left: $space-5;
+      padding: $space-2 $space-4 $space-8 !important;
+
+      border: 3px solid $color-yellow;
+    }
+
+    @media (max-width: $grid-breakpoint-large) {
+      padding: $space-3;
+    }
+
+    @media (max-width: $grid-breakpoint-extra-large) {
+      padding: $space-7;
     }
   }
 
   @media (min-width: $grid-breakpoint-small) {
     gap: 0;
   }
 
-  @include container;
+  @media (width <= 1199px) {
+    float: left;
+    flex-direction: column;
+    gap: 0 !important;
```

Comment:
Why the !important? we shouldn't need this - we're not overriding external css right?